### DISCUSSION TITLE
Is there a way to regenerate the SSL cert for the manager or use your own?

### DISCUSSION THREAD

**Discussion by ihuxn4hi** [Security-Onion-Solutions/securityonion#9686] [Unsupported Versions]
Hi there,

Is there a way to regenerate the SSL cert or use your own (like Let's Encrypt)?

Thanks!

**Comment by cm-ops** [comment]
To regenerate the SSL certificate for SOC you can run `salt-call state.apply ssl`. If you want to use your own certificate, create your .crt/.key and replace `managerssl.crt` and `managerssl.key` in` /etc/pki` with yours, keep the name `managerssl.crt` and `managerssl.key`, restart nginx with` so-nginx-restart`, and verify in browser you are using your cert.

**Reply by ihuxn4hi** [reply]
That worked perfectly -- thank you very much!

**Comment by adamphetamine** [comment]
if you want to use letsencrypt here's a method that works, but could probably be improved
 (ie. please let me know if you can improve it)

https://github.com/adamphetamine/letsencrypt-security-onion
